java/java-ranger-regression/alarm/impl/ALARM_Functional.java

```diff
@@ -0,0 +1,1789 @@
+
```

Comment:
nit-pick: most files in the java benchmark repository are formatted using 2 white spaces instead of tabs.

There is not strict linting yet, but you might nevertheless adhere to the repository standard.

Reply:
Sure, will do.